controllers/iprange/iprange.go

```diff
@@ -0,0 +1,91 @@
+package iprange
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+	"regexp"
+	"strings"
+
+	"github.com/apparentlymart/go-cidr/cidr"
```

Comment:
Std lib has CIDR support, do we really need this 3rd-party lib?

Reply:
these codes are reference from https://github.com/netdata/go.d.plugin/blob/v0.40.4/pkg/iprange/parse.go, but it is not satisfy our requirement, so  I made some update. as golang net pakcage can't get ip list of a CIDR. I read https://github.com/apparentlymart/go-cidr/cidr codes,  it is more efficient, otherwise the only way I can think is use a for loop to get the ip list of a CIDR.

Reply:
Consider using "github.com/mikioh/ipaddr"?

Reply:
I removed https://github.com/apparentlymart/go-cidr, now calcute CIDR range directly.